### ISSUE TITLE
Why no Angel Investor is Replying to Your Messages and Emails

### ISSUE THREAD

**Issue by ZouTiger** [ZouTiger/rupi.is#16]
---
title: 'Why no Angel Investor is Replying to Your Messages and Emails'
date: '2024-01-25'
author: rupi
slug: emailing-angel-investors
category: angel-investing
tags: [welcome]
description: How to reach out to angel investors so you get replies
updatedOn: 
---

As a founder, one of the first big challenges you face is getting funding. Many founders resort to cold emailing angel investors, but there's a problem with this approach – it's like trying to hit a bullseye in the dark. As an angel investor myself, I see many such emails, all echoing the same, “We’re building XYZ, interested?” 

I am not opposed to cold emails but there is a correct way to write these emails, which I will try to describe. 

**Case in Point: The Ineffective Pitch**

Consider an email I once received, anonymized for privacy:

> "Hi Rupi,
> I'm [Founder's Name], the Founder of [Company], a SF startup. We are building an [AirBnB for industry x]. We are confident we will revolutionize [x] and are raising a seed round. I'm also keen to expand my network and connect with someone of your experience.
> Are you interested?
> Looking forward to speaking soon,
> [Founder's Name]"

While polite, this email lacks substance. It’s a classic example of what not to do. It doesn't tell me how investing in this venture could be profitable, nor does it demonstrate any of the critical elements I and other angels look for in a company. It may work if we have had many conversations before or I know the founder well but it fails miserably as a cold email.

**Investor-Centric Approach: Power Law**

As a founder, you must understand that an angel investor's primary motive isn’t altruism; it’s ROI. Most angel investments go to 0 and a few will account for most of the returns of any angel - this is also true for VC funds. This is called the Power Law.

There's a certain irony here – the best way to attract investment is to show how you can make the investor a lot of money. Your email isn't just a message; it's a value proposition. Believe me, every angel wants you to succeed wildly because that is the only way they make money. 

**The Anatomy of a Winning Pitch Email**

When you write to an investor, think of yourself as a storyteller. You don't have to include all the information I am listing below. Just keep in mind you’re not just sharing information; you’re crafting a narrative and the less information you can craft it with, the better it is. 

Here’s what your story should include:

1. **Social Proof:** If industry leaders are talking about you, or you have the perfect domain knowledge and experience, that's gold. Don’t bury it; showcase it. How did you gain that domain knowledge? Is there a story there? 

2. **Capability is Your Trump Card:** Investors bet on people, not companies. The right people will make **an idea** work so highlight your team's skills and past successes. Especially in early stage, the ideas will change. Make me believe in you.

3. **Traction is Your Best Argument:** No one can argue with real, measurable traction. Show me growth, user engagement, revenue – these are the metrics that matter. Make sure you aren't focusing on vanity metrics. Don't have customers yet? How many potential users have you spoken to? How many have asked to be kept in loop? Better yet, how many have prepaid for the product? I have mixed feelings about LOIs without any skin in the game. In some sectors (enterprise) they can be a good sign but not always. 

4. **Who Else is Betting on You?:** If other investors, especially known ones, are in, it’s a strong endorsement. Highlight whether they are investing in the new round or invested in past rounds. If they have invested in you multiple times; even better. 

5. **Tailor Your Pitch:** A generic pitch is forgettable. Research your investor, understand what drives them, and personalize your pitch accordingly. For example, I love to learn new things so is there something you have learned in your customer discovery that is counter-narrative? Include that in your email. 

6. **Include a Deck and/or a demo:** Consider including a PDF with background information or a link to a video demo.  

7. **Make Yourself Easy to Find:** Include links that are relevant such as to your landing page, any media coverage, any social media buzz etc. Make accessing information as easy as a click. I have received messages on LinkedIn where I have to open the message, click on the sender, look at the company name, search for the company on Google, then go to the website of the company - annoying!

**Investors’ Desire for Your Success**

Think of your fundraise itself as a product and reduce friction in any way you can. 

Remember, investors want you to succeed. It’s not just about the money. We're rooting for a thriving startup ecosystem. But you need to make a strong case for why your startup will be among the successful few. Your communication, your vision, and your evidence of potential success are crucial in all of that.

**If You Don’t Have a Product Yet**

If you're yet to build a product, your first step should be to focus on that. Use Figma to do a few designs. Use those to talk to potential customers. You can do a lot with a Figma prototype. 

Fundraising on just an idea doesn't really work anymore (unless you have a good track record and/or other exits). I do admit that in some cases, you can't build a product with raising a lot of money. In those cases, cold outreach to angel investors is probably not the best route for you.

**In Conclusion**

Raising funds from angels isn’t about casting a wide net with generic emails. It’s about storytelling, about demonstrating value, and about making a compelling case for your startup. Be detailed, be passionate, and most importantly, be convincing. Remember, in the world of startups, as in writing, it’s often the small details that make the biggest impact.
